net/dpdk/Makefile

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME := dpdk
+PKG_VERSION := 21.05
+
+PKG_SOURCE := dpdk-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL := http://static.dpdk.org/rel
+PKG_MD5SUM := a78bba290b11d9717d1272cc6bfaf7c3
```

Comment:
PKG_HASH

Reply:
@neheb ..Thanks for the comment..Changed to PKG_HASH